pkgs/development/tools/tradcpp/aarch64.patch

```diff
@@ -0,0 +1,12 @@
+diff a/config.h b/config.h
+--- a/config.h
++++ b/config.h
+@@ -124,6 +124,8 @@
+ #define CONFIG_CPU "__ppc64__"
+ #elif defined(__ARM__)
+ #define CONFIG_CPU "__ARM__"
++#elif defined(__aarch64__)
++#define CONFIG_CPU "__aarch64__"
```

Comment:
I am sure netbsd would also accept this as a patch.

Reply:
@Mic92 Why do you think so?

Reply:
@Mic92 Sorry, I think I misread your comment. I agree, the patch is reasonable, so they probably will accept it.

Reply:
I have sent it to dholland, the maintainer.

Reply:
I got a response:

```
On Fri, Dec 07, 2018 at 10:51:46AM +0000, J?rg Thalheim wrote:
 > This has been added in https://github.com/NixOS/nixpkgs/pull/51585

Several people sent this in more or less at once... anyway I just
posted a new tarball (0.5.3) with this and a few other similar bits.

Thanks.
```

Reply:
@Mic92 The other person must be me.